payments-core/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt

```diff
@@ -1025,12 +1036,14 @@ data class PaymentMethodCreateParams internal constructor(
         @JvmOverloads
         fun createUSBankAccount(
             billingDetails: PaymentMethod.BillingDetails? = null,
-            metadata: Map<String, String>? = null
+            metadata: Map<String, String>? = null,
+            allowRedisplay: PaymentMethod.AllowRedisplay? = null,
```

Comment:
Why not add it to sepa debit? Why not add it to all LPMs? Which LPMs support it?

Reply:
After talking with John, `allow_redisplay` can be changed for all LPMs. Added for all LPMs.